src/utils.ts

```diff
@@ -85,8 +85,13 @@ export async function subgraphRequest(url: string, query, options: any = {}) {
 }
 
 export function getUrl(uri) {
```

Comment:
Can we add possibility to edit the gateway like this?
```suggestion
export function getUrl(uri, gateway?) {
```

Reply:
Like that?

Reply:
Yes